src/pages/html/html-entities/index.md

```diff
@@ -3,7 +3,7 @@ title: HTML Entities
 ---
 ## HTML Entities
 
-In HTML characters like ```<```, ```>``` has special meaning. For example, if you use ```<``` character, browser will parse it as a beginning of an HTML element tag. Instead you need to use HTML entity&mdash;a string beginning with ```&``` and ending with ```;```. And for any special symbol there is an HTML entity.
+In HTML characters like ```<```, ```>``` has special meaning. For example, if you use the ```<``` character, your browser will parse it as a beginning of an HTML element tag. Instead you need to use an HTML entity&mdash;a string beginning with ```&``` and ending with ```;```. For any special symbol there is an HTML entity.
```

Comment:
Grammatical fix "In HTML characters like" to "HTML characters like"
Grammatical fix "has special meaning" to "**have** special meaning"